cmd/ebrelayer/main.go

```diff
@@ -171,27 +175,30 @@ func RunEthereumRelayerCmd(cmd *cobra.Command, args []string) error {
 	}
 
 	// Get the validator's name and account address using their moniker
-	validatorAccAddress, validatorName, err := sdkContext.GetFromFields(validatorFrom, false)
+	validatorAccAddress, validatorName, err := context.GetFromFields(inBuf, validatorFrom, false)
 	if err != nil {
 		return err
 	}
 	// Convert the validator's account address into type ValAddress
 	validatorAddress := sdk.ValAddress(validatorAccAddress)
 
-	// Get the validator's passphrase using their moniker
-	passphrase, err := keys.GetPassphrase(validatorFrom)
-	if err != nil {
-		return err
-	}
+	passphrase := ""
+	// FIXME: use keyring
```

Comment:
@alessio do you know how to update this using the new keyring?

Reply:
You just don't need the passphrase anymore. And `MakeSignature` takes a Keybase interface instance as first argument, which will then be used to sign with the `validatorName` key, if such key exists in the keybase

Reply:
`passphrase` can be left blank as it will be ignored